browser/main/lib/ConfigManager.js

```diff
@@ -65,7 +66,15 @@ export const DEFAULT_CONFIG = {
     spellcheck: false,
     enableSmartPaste: false,
     enableMarkdownLint: false,
-    customMarkdownLintConfig: DEFAULT_MARKDOWN_LINT_CONFIG
+    customMarkdownLintConfig: DEFAULT_MARKDOWN_LINT_CONFIG,
+    prettierConfig: ` {
+      "trailingComma": "es5",
+      "tabWidth": 4,
+      "semi": false,
+      "singleQuote": true,
+      "parser":"markdown"
```

Comment:
Can you append this `parser` option at the end? Because this will never change so we should make it default.

Reply:
@ZeroX-DG  do you mean as in don't have it in the options that show up in the config and then append the parser property before use?

Reply:
Assuming i understood the comment correctly this should be addressed in #1d59d89

Reply:
Yep that's exactly what I wanted, thank you!